src/passes/pass.cpp

```diff
@@ -45,7 +45,7 @@ void PassRegistry::registerPass(const char* name,
                                 const char* description,
                                 Creator create) {
   assert(passInfos.find(name) == passInfos.end());
-  passInfos[name] = PassInfo(description, create);
+  passInfos[name] = PassInfo(description, create, false);
```

Comment:
Why is this needed? Isn't false the default?

Reply:
Indeed. This is a leftover from when I had added a second default argument. Removed.